src/sass/components/_slogan.scss

```diff
@@ -1,20 +1,38 @@
 .slogan {
   width: 230px;
-  margin-bottom: 19px;
+  margin-bottom: 20px;
+
+  @media screen and (min-width: 768px) {
+    display: flex;
+    justify-content: space-between;
+    width: auto;
+  }
 }
 
 .slogan__title {
   color: var(--color-white);
   font-family: 'Titan One', cursive;
   font-weight: 400;
-  font-size: 26px;
+  font-size: var(--font-header-label-tablet);
```

Comment:
Mogłbyś te font-size napisać za pomocą px? Chodzi o to, żeby nie używać zmiennej z mylącą nazwą

Reply:
Faktycznie, jak tylko zasiądę do kompa to to poprawie